packages/fetchye-redux-provider/rollup.config.js

```diff
@@ -0,0 +1,65 @@
+import { nodeResolve } from '@rollup/plugin-node-resolve';
```

Comment:
Can we check that all of them have licenses?

Reply:
Done